docs/tutorials/task.csv-to-oscal-cd/csv-to-oscal-cd.md

```diff
@@ -0,0 +1,151 @@
+<td>Parameter_Id
+<td>String
+<td>optional
+<td>A textual label that uniquely identifies the parameter associated with that policy (desired state) or controls implemented by the policy (desired state).	A description of the parameter including the purpose and use of the parameter.
+<td>component.control-implementation.prop["Parameter_Id"]<br>*and*<br>component.control-implementation.set-parameter.param-id
+<td>minimum_password_length
+
+<tr style="text-align:left;vertical-align:top">
+<td>Parameter_Description
+<td>String
+<td>optional
+<td>A description of the parameter including the purpose and use of the parameter.
+<td>component.control-implementation.prop["Parameter_Description"]
+<td>Minimum Password
+
+<tr style="text-align:left;vertical-align:top">
+<td>Parameter_Default_Value
+<td>String
+<td>optional
+<td>A value recommended in this profile for the parameter of the control or policy (desired state).
+<td>12
+<td>component.control-implementation.set-parameter.values
+
+<tr style="text-align:left;vertical-align:top">
+<td>Parameter_Value_Alternatives
+<td>String List (blank separated)
+<td>optional
+<td>ONLY for the policy (desired state) parameters: A value or set of values the parameter can take.
+<td>component.control-implementation.prop["Parameter_Value_Alternatives"]
+<td>12 8
+
+<tr style="text-align:left;vertical-align:top">
+<td>Check_Id
+<td>String
+<td>optional
+<td>A textual label that uniquely identifies a check of the policy (desired state) that can be used to reference it elsewhere in this or other documents.
+<td>component.control-implementation.prop["Check_Id"]
+<td>check_password_policy_min_length_characters
+
+<tr style="text-align:left;vertical-align:top">
+<td>Check_Description
+<td>String
+<td>optional
+<td>A description of the check of the policy (desired state) including the method (interview or examine or test) and procedure details.
+<td>component.control-implementation.prop["Check_Description"]
+<td>Check whether password policy requires minimum length of 12 characters
+
+<tr style="text-align:left;vertical-align:top">
+<td>Fetcher
+<td>String
+<td>optional
+<td>A textual label that uniquely identifies a collector of the actual state (evidence) associated with the policy (desired state) that can be used to reference it elsewhere in this or other documents.
+<td>component.control-implementation.prop["Fetcher"]
+<td>fetch_password_policy_min_length_characters
+
+<tr style="text-align:left;vertical-align:top">
+<td>Fetcher_Description
+<td>String
+<td>optional
+<td>A description of the collector of the actual state (evidence) associated with the policy (desired state) including the method (interview or examine or API) and questionaire
+<td>component.control-implementation.prop["Fetcher_Description"]
+<td>Fetch whether password policy requires minimum length of 12 characters
+
+<tr style="text-align:left;vertical-align:top">
+<td>Resource_Instance_Type
+<td>String
+<td>optional
+<td>not used
```

Comment:
Deprecated? Or planned to be supported in the future?

Reply:
should esource_Instance_Type be optional? instead of not used?

Reply:
This is the string that would be part of all instance ids (for this type of resource), so that it can be selected. If this is not being used then lets remove it,

Reply:
Actually, I was mistaken. Resource_Instance_Type is used. I'll fix the doc.